cassovary-core/src/main/scala/com/twitter/cassovary/graph/TestGraph.scala

```diff
@@ -278,6 +278,42 @@ object TestGraphs {
     new BipartiteGraph(leftSide, rightSide, BipartiteGraphDir.Both)
   }
 
+  def bipartiteGraphWithUniqueNodeIds = {
+
+    val leftNodes = new Array[BipartiteNode](7)
+
+    var inBounds = Array(1, 3, 5);
```

Comment:
consider not using vars. In fact no point using intermediate variables inBounds and outBounds. Just directly use as in new LeftNode(2, Array(...), Array(...))